README.md

```diff
@@ -10,20 +10,70 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
 
 ![screen shot 2016-04-12 at 12 22 02 pm](https://cloud.githubusercontent.com/assets/4596845/14469165/5d27495c-00b1-11e6-9d28-327938463adf.png)
 
-## Quick start
 
-1. Install Go 1.6, and ensure your `GOPATH` is set. Using [gvm](https://github.com/moovweb/gvm) is recommended.
-1. Install the tool
-
-    ```bash
-    go get github.com/opencontrol/compliance-masonry
-    ```
-
-1. Run the CLI
+## Installation
+Compliance Masonry is packaged into a downloadable executable program for those
+who want to use Compliance Masonry without the need to install any external
+dependencies or programs. This is extremely beneficial for those who may to
+jump right in without worrying about much technical setup.
+
+### Mac OS X Installation
+#### Technical Users - Mac OS X
+In your terminal, run the following:
+```sh
+curl -L https://github.com/opencontrol/compliance-masonry/releases/download/v1.1.2/compliance-masonry_1.1.2_darwin_amd64.zip -o compliance-masonry.zip
+unzip compliance-masonry.zip
+cp compliance-masonry_1.1.2_darwin_amd64/compliance-masonry /usr/local/bin
+```
 
-    ```bash
-    compliance-masonry
-    ```
+#### Less Technical Users - Mac OS X
+1. Go to the [Github Releases
+Page](https://github.com/opencontrol/compliance-masonry/releases).
+2. Download the package that corresponds to your machine and operating system.
+  - The following are common file endings for common systems configurations to
+  help you decide which one.
+  - For Mac OS X, you'll want the file ending `_darwin_amd64.zip`
+3. Double-click on the downloaded package to unzip the archive and it
+should result in a folder which contains a file called `compliance-masonry`.
+4. Right click on `compliance-masonry` and select `Copy`
+5. Open `Finder`
+6. In the navigation bar, click `Go` -> `Go to Folder...`
+7. Type in `/usr/local/bin` to open the `bin` folder
+8. In the navigation bar, click `Edit` -> `Paste Item`
```

Comment:
While I appreciate the sentiment behind this section, the reality is that users are going to be running Masonry from the command line anyway, so if they aren't comfortable with running those handful of commands above, they are going to have a hard time with the rest. Therefore, I would take this section out, for simplicity.